codec-http/src/main/java/io/netty/handler/codec/rtsp/RtspEncoder.java

```diff
@@ -50,22 +49,19 @@ protected void encodeInitialLine(final ByteBuf buf, final HttpMessage message)
            throws Exception {
         if (message instanceof HttpRequest) {
             HttpRequest request = (HttpRequest) message;
-            HttpHeaders.encodeAscii(request.method().toString(), buf);
+            buf.writeCharSequence(request.method().toString(), CharsetUtil.US_ASCII);
```

Comment:
instead of doing `toString()` did you consider using `asciiName()` and copying the bytes directly?